src/components/Layout/layout.scss

```diff
@@ -10,7 +10,7 @@
 .layout-tables {
     display: flex;
     flex: 1;
-    flex-Direction: row;
+    flex-Direction: column;
     justify-content: space-between;
     gap: 20px;
```

Comment:
I noticed you're a bit inconsistent with gaps and px.  You should consider switching to rem here (lots of data available online, a quick google gives:
https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15)